ThermiaOnlineAPI/const.py

```diff
@@ -103,3 +103,11 @@
 ###############################################################################
 
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+###############################################################################
+# calendar functions
+###############################################################################
+CAL_FUNCTION_REDUCED_HEATING_EFFECT = "15001"
+CAL_FUNCTION_HOT_WATER_BLOCK= "15002"
+CAL_FUNCTION_SILENT_MODE = "15003"
+CAL_FUNCTION_EVU_MODE = "15004"
```

Comment:
This is fine to have as supported functions by the Python library, but they should also be fetched from the API and the supported list should be returned as a property by heat pump because not all heat pumps will support all the functions.